repl/repl.go

```diff
@@ -99,19 +118,19 @@ func (r *repl) createAccount() {
 		emptyComplete,
 		prompt.OptionPrefixTextColor(prompt.LightGray))
 
-	err := r.node.CreateAccount(generatePassphrase, accountInfo)
+	err := r.client.CreateAccount(generatePassphrase, accountInfo)
 	if err != nil {
-		r.node.Debug(err.Error())
+		log.Debug(err.Error())
 	} else {
```

Comment:
In go a good pattern is,
```go
if err != nil {
   log.Error(err)
   retutn
}
r.setup()

```

Instead of using if/else just return inside the error condition